core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorAdapterTest.scala

```diff
@@ -58,7 +62,7 @@ class GroupCoordinatorAdapterTest {
   @ApiKeyVersionsSource(apiKey = ApiKeys.JOIN_GROUP)
   def testJoinGroup(version: Short): Unit = {
     val groupCoordinator = mock(classOf[GroupCoordinator])
-    val adapter = new GroupCoordinatorAdapter(groupCoordinator)
+    val adapter = new GroupCoordinatorAdapter(groupCoordinator, Time.SYSTEM)
```

Comment:
Curious if we wanted to mock time here. Doesn't seem to be a big difference one way or the for most of these since they don't use a time component anyway, but could be a good practice to have a MockTime. (Looks like we do use it for the test added 😄 )

Reply:
Maybe I'm just recalling some recent tests where I used system time because it was already there and it led to flakiness. 
I suppose here though it is easier to switch so maybe not a problem.

Reply:
Yeah. In this case, it does not matter because the test does not depend on time at all.